specification/ai/data-plane/Azure.AI.Projects/stable/v1/azure-ai-projects.json

```diff
@@ -1090,6 +1090,25 @@
         "blobReference"
       ]
     },
+    "AssistantMessage": {
```

Comment:
we should not add these APIs to v1 since we know that breaking changes are coming